package.json

```diff
@@ -40,28 +40,25 @@
     },
     "dependencies": {
         "@azure/functions": "^1.2.3",
-        "@types/lodash": "^4.14.119",
-        "@types/node": "10.14.2",
-        "@types/uuid": "~3.4.4",
-        "@types/validator": "^9.4.3",
         "axios": "^0.21.1",
-        "commander": "~2.9.0",
         "debug": "~2.6.9",
         "lodash": "^4.17.15",
-        "rimraf": "~2.5.4",

```

Comment:
Only one reference found in a package.json script. However, these scripts are never run in production so `rimraf` can safely be a dev dependency.
<img width="1108" alt="Screen Shot 2022-04-22 at 2 45 34 PM" src="https://user-images.githubusercontent.com/11282622/164802208-366b9cf6-5018-4e94-bb55-631d0a59ac09.png">